tests/features/common/kie-controller-common.feature

```diff
@@ -15,13 +15,22 @@ Feature: KIE Controller configuration common tests
       | KIE_ADMIN_USER             | customAdm     |
       | KIE_ADMIN_PWD              | custom" Adm!0 |
       | KIE_SERVER_TOKEN           | token         |
-    Then file /opt/eap/standalone/configuration/application-users.properties should not contain customAdm
-     And file /opt/eap/standalone/configuration/application-roles.properties should not contain customAdm
-     And container log should contain -Dorg.kie.server.token=token
+    Then file /opt/eap/standalone/configuration/application-users.properties should contain customAdm=a4d41e50a4ae17a50c1ceabe21e41a80
+     And file /opt/eap/standalone/configuration/application-roles.properties should contain adminUser=kie-server,rest-all,admin,kiemgmt,Administrators,user
```

Comment:
This test scenario does not look correct now. Before there was checked that the token is used for communication instead of user. Now user is created.
In this case token should be still used.

Maybe question related to whole RHPAM-2175 - Do we still need to have tokens in images / templates ?

Reply:
Now, if I am not wrong, the token and the username/password will be accepted. That's why the user is also being created.

The question is, does the token make sense when no sso/ldap is set?